pkg/commands/os.go

```diff
@@ -75,33 +67,15 @@ func sanitisedCommandOutput(output []byte, err error) (string, error) {
 	return outputString, nil
 }
 
-func getPlatform() *Platform {
-	switch runtime.GOOS {
```

Comment:
Here we have a default platform. Where is the logic in the PR to support defaulting? I can see that there is a file for linux/darwin/windows but what happens if there is another OS outside of that set?

Reply:
So know we have a default platform file with a no build directive for windows and a windows platform specific file

Reply:
I added a compile step to circle to catch this kind of error

Reply:
something failing with darwin arm

Reply:
ok so as I can see from build done when releasing a new version there is release for darwin arm so I simply excluded it from building